src/pkg/packaging/dir.proj

```diff
@@ -81,14 +81,16 @@
       <templateSvg>$(ProjectDir)resources/images/version_badge.svg</templateSvg>
     </PropertyGroup>
 
+    <MakeDir  Condition="!Exists('$(BaseOutputRootPath)')" Directories="$(BaseOutputRootPath)" />
+    
     <WriteLinesToFile
       File="$(OutputVersionBadge)"
       Lines="$([System.IO.File]::ReadAllText('$(templateSvg)').Replace('ver_number', '$(ProductVersion)'))"
       Overwrite="true"
                       />
   </Target>
   
-  <Target Name="GenerateCompressedFiles" DependsOnTargets="InitPackage;GenerateZip;GenerateTarBall" />
+  <Target Name="GenerateCompressedFiles" DependsOnTargets="InitPackage;GenerateZip;GenerateTarBall" Condition="'$(UsePrebuiltPortableBinaries)' == 'false'"/>
```

Comment:
This sounds fine to me as long as there is a single portable build, output of which you are consuming for building the platform installers.